src/pages/preferences/preferencesTabs/NewUITab.tsx

```diff
@@ -0,0 +1,74 @@
+import React from "react";
+import { useMutation } from "@apollo/react-hooks";
+import { Body } from "@leafygreen-ui/typography";
+import Toggle from "@leafygreen-ui/toggle";
+import styled from "@emotion/styled";
+import Card from "@leafygreen-ui/card";
+import { useBannerDispatchContext } from "context/banners";
+import { UPDATE_USER_SETTINGS } from "gql/mutations/update-user-settings";
+import {
+  UseSpruceOptions,
+  UpdateUserSettingsMutation,
+  UpdateUserSettingsMutationVariables,
+} from "gql/generated/types";
+
+interface NewUITabProps {
+  useSpruceOptions: UseSpruceOptions;
+}
+export const NewUITab: React.FC<NewUITabProps> = ({ useSpruceOptions }) => {
+  const { spruceV1 } = useSpruceOptions;
+  const dispatchBanner = useBannerDispatchContext();
+  const [updateUserSettings, { loading }] = useMutation<
+    UpdateUserSettingsMutation,
+    UpdateUserSettingsMutationVariables
+  >(UPDATE_USER_SETTINGS, {
+    onCompleted: () => {
+      dispatchBanner.successBanner(
```

Comment:
Controls that save data immediately typically do not give any feedback about it because it's a bit jarring/annoying, and I think that applies here as well. It's definitely weird that some of the tabs have a confirmation button and some don't, but I think the pattern is clear enough here to remove the success banner

Reply:
All the other preferences tabs do show a success banner when ever we interact with them. and perform some type of modification.

Reply:
Right, but those have a confirmation button. This one doesn't

Reply:
I personally like seeing the confirmation banner but I also think this is a good question for design

Reply:
I'll open a ticket